common/stream.h

```diff
@@ -553,173 +628,217 @@ class ReadStream : virtual public Stream {
-
+	/** @} */
 };
 
-
 /**
- * Interface for a seekable & readable data stream.
+ * Interface for a seekable and readable data stream.
  *
- * @todo Get rid of SEEK_SET, SEEK_CUR, or SEEK_END, use our own constants
+ * @todo Get rid of SEEK_SET, SEEK_CUR, or SEEK_END, use our own constants.
  */
 class SeekableReadStream : virtual public ReadStream {
 public:
 
 	/**
-	 * Obtains the current value of the stream position indicator of the
-	 * stream.
+	 * Obtain the current value of the stream position indicator.
 	 *
-	 * @return the current position indicator, or -1 if an error occurred.
+	 * @return The current position indicator, or -1 if an error occurred.
 	 */
 	virtual int32 pos() const = 0;
 
 	/**
-	 * Obtains the total size of the stream, measured in bytes.
-	 * If this value is unknown or can not be computed, -1 is returned.
+	 * Obtain the total size of the stream, measured in bytes.
+	 * If this value is unknown or cannot be computed, -1 is returned.
 	 *
-	 * @return the size of the stream, or -1 if an error occurred
+	 * @return The size of the stream, or -1 if an error occurred.
 	 */
 	virtual int32 size() const = 0;
 
 	/**
-	 * Sets the stream position indicator for the stream. The new position,
-	 * measured in bytes, is obtained by adding offset bytes to the position
-	 * specified by whence. If whence is set to SEEK_SET, SEEK_CUR, or
-	 * SEEK_END, the offset is relative to the start of the file, the current
-	 * position indicator, or end-of-file, respectively. A successful call
-	 * to the seek() method clears the end-of-file indicator for the stream.
+	 * Set the stream position indicator for the stream.
+	 *
+	 * The new position, measured in bytes, is obtained by adding offset bytes
+	 * to the position specified by whence. If whence is set to SEEK_SET, SEEK_CUR,
+	 * or SEEK_END, the offset is relative to the start of the file, the current
+	 * position indicator, or end-of-stream, respectively. A successful call
+	 * to the seek() method clears the end-of-stream indicator for the stream.
 	 *
-	 * @note The semantics of any implementation of this method are
-	 * supposed to match those of ISO C fseek().
+	 * @note The semantics of any implementation of this method is
+	 * supposed to match that of ISO C fseek().
 	 *
-	 * @param offset	the relative offset in bytes
-	 * @param whence	the seek reference: SEEK_SET, SEEK_CUR, or SEEK_END
-	 * @return true on success, false in case of a failure
+	 * @param offset	Relative offset in bytes.
+	 * @param whence	Seek reference: SEEK_SET, SEEK_CUR, or SEEK_END.
+	 *
+	 * @return True on success, false in case of a failure.
 	 */
 	virtual bool seek(int32 offset, int whence = SEEK_SET) = 0;
 
 	/**
-	 * TODO: Get rid of this??? Or keep it and document it
-	 * @return true on success, false in case of a failure
+	 * Skip the given number of bytes in the stream.
+	 * 
+	 * This is equivalent to calling:
+	 * @code
+	 * seek(offset, SEK_CUR)
```

Comment:
This should be `SEEK_CUR`, not `SEK_CUR` (sorry that was a typo in my comment).
I will still merge and then fix the typo in tree.